playbooks/test.yml

```diff
@@ -21,18 +21,97 @@
 
   tasks:
   - block:
+    # - name: copy java program to zos
```

Comment:
Should this test playbook be added to `build_ignore` in galaxy.yml so it doesn't built into the collection? 

Interesting I just checked the previous release and this file existed before also but didn't get added to the built collection. Do you know how this being ignored?

Reply:
maybe ignored by CICD?

Reply:
It'd be good for us to confirm. I also noticed that some other files/playbooks are also being ignored, perhaps intentionally (for ex: playbooks/delete-ims-datasets.yml)